templates/book.html

```diff
@@ -16,20 +16,25 @@
             <div class="header-items"><a href="{{ url_for('users') }}">Users</a></div>
         </nav>
 
-        <div>
-                <div class="single-book">
-    
-                    <!-- One parent -->
-                    <div class="single-book-image"><img src="{{ book['image'] }}" alt="{{ book['title'] }}"></div>
-                
-                    <!-- Second parent with book details -->
-                    <div class="display-single-book">
-                        <div class="space single-book-title">{{ book['title'] }}</div>
-                        <div class="space single-book-author"><br>By: {{ book['author'] }}</div>
-                        <div class="space single-book-summary text"><br>{{ book['summary'] }}</div>
-                    </div>
-                </div>
+    <div>
+        {% for book in books %}
```

Comment:
Instead of passing all the `books` back to this template in your view function, you can have your query on https://github.com/amymhaddad/upcase_booklist_app/pull/10/files#diff-3f41e546893dc64b71aaacad12cad815R51

return a single book by using `filter` with `one`. See the documentation here:

https://docs.sqlalchemy.org/en/14/orm/tutorial.html#returning-lists-and-scalars

Then you won't need to do a loop and apply a filter here: `{% if book.id == book_id %}`